.ci/scripts/pre_archive_test.py

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env python3
+
+import os
+import distutils
+from distutils import dir_util
```

Comment:
Is there a specific reason you want to use `distutils`? It is considered to be a legacy tool has an uncertain future with `setuptools` being the [preferred user-accessible library](https://docs.python.org/3.8/library/distutils.html).

Reply:
For using `distutils.dir_util.copy_tree`, I did try using some other libraries but that one did let me keep up if the folder already exists. The ones that I gave a go was: [shutil.copytree](https://docs.python.org/3/library/shutil.html#shutil.copytree)

In order to use `distutils.dir_util.copy_tree` I had to declare the `import distutils ` and `from distutils... `otherwise I faced issues like [shown here](https://stackoverflow.com/a/33417555)

I'm quite newbie with python, so if the above is a security/legacy flag, regardless of what I just said please let me know

Reply:
AFAIK it's not a security concern so it should be fine.